src/components/Dropdown/asyncDropdown/AsyncDropdown.jsx

```diff
@@ -112,6 +116,7 @@ class AsyncDropDown extends React.Component {
       multiSelect,
       selectedItems,
       disableSelected,
+      onCreate,
```

Comment:
AsyncDropdown bør ha en regexp som kun lar deg opprette dersom det som skrives inn har korrekt format. Defaultverdi kan være en regexp som slepp gjennom alt for at ikkje dette skal knekke andre plasser.

Reply:
Ønsker du noe ekstra funksjonalitet når bruker ikke får lov å opprette kompetansemålkode med feil format? f.eks rød boks rundt input-feltet eller tilbakemelding på hvorfor det ikke går? eller skal det bare ikke skje noe når formatet er feil?